tests/wpunit/PostObjectMutationsTest.php

```diff
@@ -8,8 +8,34 @@ class PostObjectMutationsTest extends \Codeception\TestCase\WPTestCase {
 	public $admin;
 	public $subscriber;
 	public $author;
+	public $attachment_id;
+	public $media_item_id;
+	public $post;
+	public $post_uid;
+	public $create_media_item_variables;
+	public $altText;
+	public $authorId;
+	public $caption;
+	public $commentStatus;
+	public $current_date_gmt;
+	public $date;
+	public $dateGmt;
+	public $description;
+	public $filePath;
+	public $fileType;
+	public $slug;
+	public $status;
+	public $pingStatus;
+	public $parentId;
 
 	public function setUp() {
+
+		/**
+		 * Clear the uploads folder
+		 */
+		$_wp_content_upload_dir = WP_CONTENT_DIR . '/uploads';
```

Comment:
this seems like it could catch some people off-guard if they aren't running tests in isolation. call out in docs?